src/main/java/org/folio/innreach/domain/service/impl/ContributionServiceImpl.java

```diff
@@ -151,4 +158,32 @@ private MappingValidationStatusDTO validateInnReachLocations(UUID centralServerI
       .collect(Collectors.toList());
   }
 
+  @Override
+  public void startInitialContribution(UUID centralServerId) {
+    var contribution = createEmptyContribution(centralServerId);
+    repository.save(contribution);
+
+    var request = createInstanceIterationRequest();
+    log.info("Calling mod-inventory storage...");
+    client.startInitialContribution(request);
```

Comment:
probably this call should be commented until the endpoint is implemented on mod-inventory-storage side

Reply:
Once contribution started and response received from mod-inventory-storage, job id should be stored in `Contribution` entity. Please extend the model and save job id in the entity